portal.go

```diff
@@ -1966,16 +1976,21 @@ func (portal *Portal) handleNormaliMessage(msg *imessage.Message, dbMessage *dat
 		portal.log.Warnfln("iMessage %s doesn't contain any attachments nor text", msg.GUID)
 	}
 	for index, converted := range parts {
+		if mxID != nil {
+			converted.Content.SetEdit(*mxID)
+		}
```

Comment:
This will do the wrong thing if there are multiple parts

Reply:
I was not the most confident in this code. Do you have anything I can reference to improve it? Maybe I should skip edits entirely if there is more than one part? I don't know how apple handles multipart message edits.